app/interactors/update_project/update_record.rb

```diff
@@ -0,0 +1,19 @@
+class UpdateProject
+  class UpdateRecord
+    include Interactor
+
+    delegate :prepared_project_params, to: :context
+
+    def call
+      context.project = project
+      context.project.assign_attributes(prepared_project_params)
+      context.fail!(error: "Invalid data") unless context.project.save
+    end
+
+    private
+
+    def project
+      @project ||= Project.find_by!(prepared_project_params[:id])
```

Comment:
It never has `id` key. Have you even run this code?

Reply:
I run this code, but didn't test well